### PR TITLE
Adds defaults for AWS resources

### PR DESCRIPTION
**Pull request by scottford-io**
image

This PR adds a first pass at sane defaults for AWS resources. Wherever possible, I defaulted to the `arn` since it has a bunch of information like the account number, region, and the name of the resource. 

Signed-off-by: Scott Ford <scott@scottford.io>

### CODE REVIEW

**Comment by scottford-io** [comment]
I have read the Mondoo CLA Document and I hereby sign the CLA

File: resources/packs/aws/aws.lr [hunk 1]
```diff
@@ -3,7 +3,7 @@ import "../core/core.lr"
 option go_package = "go.mondoo.com/cnquery/resources/packs/aws"
 
 // AWS Resource
-aws {
+aws @defaults("aws.account.id") {
```
Comment (arlimus) [inline resources/packs/aws/aws.lr]: let's simplify this one to `@defaults("account.id")`

**Comment by preslavgerchev** [comment]
I believe the json files needs updating to add the defaults there too (so they can get picked up later when executing the query), `make cnquery/generate` should do the job

**Comment by arlimus** [comment]
Thank you for catching that @preslavgerchev !!

**Pull request merged**